mail_service/worker_report_scan_results.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+import beanstalkc
+import json
+import subprocess
+import re
+import time
+
+bs = beanstalkc.Connection(host="172.17.0.1")
+bs.watch("report_scan_results")
+
+while True:
+  print "==> Listening to report_scan_results tube"
+  job = bs.reserve()
+  job_id = job.jid
+  job_info = json.loads(job.body)
+  print "==> Received job id= %s for reporting scan results" % job_id
+
+  print "==> Retrieving message details.."
+  notify_email = job_info['notify_email']
+
+  # find image's full name and append the desired tag
+  image_under_test = job_info.get('name').split(":")[0] + ":" \
+    + job_info.get("tag")
+  print "==> Image under test is %s" % image_under_test
+
+  subject = "Scanning results for image: %s" % image_under_test
+  email_body = compose_email_text(job_details)
+
+  print "==> Sending scan results to user %s " % notify_email
+  command = "/mail_service/send_mail.sh"
+  subprocess.call([
+      command,
+      subject,
+      email_body,
+      notify_email,
+      ])
```

Comment:
After sending scan results to user, lets relay the job to next phase, i.e. delivery phase